src/components/AutoAdvanceToggle/AutoAdvanceToggle.js

```diff
@@ -19,14 +19,13 @@ const AutoAdvanceToggle = ({ label = "Autoplay", showLabel = true }) => {
           {label}
         </span>
       )}
-      <label className="ramp--auto-advance-toggle"
-        aria-labelledby='auto-advance-toggle-label'
-      >
+      <label className="ramp--auto-advance-toggle">
```

Comment:
In WAVE removal of `aria-labelledby` gave the following error;
![accessibility-wave](https://github.com/samvera-labs/ramp/assets/1331659/bea3e0ef-5409-4969-92a6-15bd5dd3473f)

Reply:
I changed the label element to a div and SiteImprove and WAVE seem happy with it.